polynote-kernel/src/main/scala/polynote/messages/Messages.scala

```diff
@@ -74,12 +74,15 @@ object NotebookCell {
 
 final case class NotebookConfig(
   dependencies: Option[DependencyConfigs],
-  repositories: Option[TinyList[RepositoryConfig]]
+  repositories: Option[TinyList[RepositoryConfig]],
+  sparkConfig: Option[ShortMap[String, String]]
```

Comment:
Now that I look at it again, I wonder whether people are really going to need to add more than 256 values to their spark conf...

Reply:
It's just one byte, and this message won't be sent very often.